cmd/snap-seccomp/main.go

```diff
@@ -698,6 +698,9 @@ func preprocess(content []byte) (unrestricted, complain bool) {
 			unrestricted = true
 		case "@complain":
 			complain = true
+		case "@squash":
```

Comment:
this simply adds a new rule which operates on the profile as a whole, it doesn't enable per-syscall denial silencing which is more useful I think, in order to actually be able to silence per-syscall denials the program we generate needs to be modified I think

Reply:
thanks for the review. I completely misunderstood how it works :-p